ladybug/skymodel.py

```diff
@@ -0,0 +1,205 @@
+"""Functions for computing radiation for different idealized skies"""
+import math
+
+
+"""ORIGINAL AHSRAE CLEAR SKY SOLAR MODEL"""
+# apparent solar irradiation at air mass m = 0
+monthly_a = [1202, 1187, 1164, 1130, 1106, 1092, 1093, 1107, 1136,
```

Comment:
Since these are Constants a good practice is to have them all caps: MONTHLYA. Also since you don't want someone to change them I would change them to be a tuple and not a list. Same comments about MONTHLYB.